migrations/sql/V2021.04.13_0628__WorkflowRun.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE orderlyweb_workflow_run
+(
+    id         INTEGER PRIMARY KEY,
```

Comment:
Would this primary key auto increment without stating it ?

Reply:
Yep, we use the same approach for report running, see https://sqlite.org/autoinc.html for details